src/pretix/control/templates/pretixcontrol/pdf/index.html

```diff
@@ -414,6 +426,24 @@ <h2>{% trans "Welcome to the PDF ticket editor!" %}</h2>
                                     </button>
                                 </div>
                             </div>
+                            <label>&nbsp;</label><br>
+                            <div class="btn-group btn-group-justified" id="toolbox-verticalalign">
```

Comment:
I think this is missing a `textcontainer` class as these controls should not be shown for old textarea elements?
```suggestion
                            <div class="btn-group btn-group-justified textcontainer" id="toolbox-verticalalign">
```